libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/Microsoft.Bot.Builder.Adapters.Slack.csproj

```diff
@@ -5,10 +5,14 @@
     <PackageVersion Condition=" '$(PackageVersion)' == '' ">4.0.0-local</PackageVersion>
     <PackageVersion Condition=" '$(PackageVersion)' != '' ">$(PackageVersion)</PackageVersion>
     <Configurations>Debug;Release;Debug - NuGet Packages</Configurations>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
This is already in master version. Check if you have the latest version of the adapters.